articles/variables-data-types-and-control-statements-in-c#/index.md

```diff
@@ -0,0 +1,339 @@
+In this article, we are going to take a look at variables, data types, and control statements in C#.
+
+### Variables in C#
+
+A variable is a storage location paired with a name, which contains some known or unknown data referred to as a value.
+
+**Syntax for declaring a variable**:
+
+```C#
+<data type> <variable_name> = value;
+```
+
+Data types are used to classify the various types of variables and state the size of data the variable is supposed to store.
+
+The different data types are:
+
+### 1. Byte
+
+The `byte` data type is used to store whole numbers from -128 to 127.
+
+### 2. Int
+
+The `int` (integer) data type is used to store whole numbers from -2,147,483,648 to 2,147,483,647.
+
+### 3. Long
+
+The `long` data type allows you to store whole numbers from -923,372,036,854,775,808 to 9,223,372,036,854,775,807.
+We should add a letter **L** at the end of the long number to specify it is a long number.
+
+### 4. Float
+
+The `float` is used to store a fractional number and a decimal number of up to 6 decimal digits. We should add an **F** at the end of the decimal number to specify that it is a `float`.
+
+### 5. Double
+
+The `double` data type stores a fraction and a decimal number of up to 15 decimal digits.
+
+*Note*: Double data type stores many decimal numbers than float. It is advisable to use double in most of the calculations.
+
+### 6. Char
+
+The `char` type is used to store **one** character.
+
+### 7. String
+
+The `string` data type is used to store **many** characters.
+
+### 8. Bool
+
+The `bool` data type is used to store **true** or **false** value.
```

Comment:
Hi @Jeff-mwangi 

The example here was good before. Please add them back but don't add the `Console.Writeline` in every example.
Also, instead of marking all the different types as headings, you can mark them as bullet points.